config/update-strategy-overlay.yml

```diff
@@ -0,0 +1,63 @@
+#@ load("@ytt:overlay", "overlay")
+#@ load("@ytt:data", "data")
+
+#@ def matcher():
```

Comment:
I tried to cover all the values from [here](https://github.com/vmware-tanzu/community-edition/blob/main/addons/packages/kapp-controller/0.44.1/bundle/config/values.yaml), but should we include these or expect users to add overlays for these?